src/unexpected_isaves/save_image.py

```diff
@@ -1,3 +1,4 @@
+from typing import List
```

Comment:
```suggestion
```

Reply:
Would you please add List to the import on line 5?

Reply:
Not sure what would be the correct flow, but since this is an important patch I'll merge this and do it myself. Patched version will be release tonight.
Thanks again for noticing this error and contributing!